CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v1.12.3
```

Comment:
I think this should be `v1.13.0`. Also, the first paragraph needs to be updated to Buildroot 2020.08 and OTP 23.1.1.